scripts/init.js

```diff
@@ -93,6 +93,7 @@ export const onLoadComponents = {
   ],
 
   async init() {
+    await this.setupTemplateAndAutoBlocks();
```

Comment:
For templates we could still use the web-component to have the same features as a component where we can have different settings by viewport and css file if needed plus the ability to use the config file option. 
It could even handle things like the placeholder render/replacement on content outside of blocks.

But this simple approach is also useful where we have just a small change like this and we don't need a full component. 
Luckily our component loader can also work with a function if the module exports a default one instead of a class.

WDYT?

We can keep it as is, add the topic for spec/refinement to explore the options.